docs/admin/system-information.rst

```diff
@@ -829,42 +829,66 @@ flag::
 Description of checked node settings
 ------------------------------------
 
+.. raw:: html
+
+  <span id="recovery-expected-nodes"></span>
+  
 Recovery expected data nodes
 ............................
 
-The check for the
-:ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting checks
-that the number of data nodes that should be waited for the immediate
-cluster state :ref:`recovery <gloss-shard-recovery>`. This value must be equal
-to the maximum number of data nodes in the cluster.
+This check looks at the 
+:ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting and 
+checks if the setting matches the actual number of data nodes present in the 
+cluster. 
+The value of this setting is compared against 
+:ref:`gateway.recover_after_data_nodes
+<gateway.recover_after_data_nodes>` (or the deprecated 
+:ref:`gateway.recover_after_nodes <gateway.recover_after_nodes>`)
+during startup to decide if the cluster can start with 
+:ref:`recovery <gloss-shard-recovery>`.
+If the actual number of nodes is below the expected number of nodes, 
+this is flagged to indicate some nodes are down. If the actual number
+is greater, this is flagged to indicate the setting should be updated.
 
 .. NOTE::
 
    For backward compatibility, setting the deprecated
    :ref:`gateway.expected_nodes <gateway.expected_nodes>` instead is still
-   supported. It must then be equal to the maximum of data and master nodes.
+   supported. It counts all nodes, not only 
+   :ref:`data-carrying nodes <node.data>`.
+
+.. raw:: html
+
+  <span id="recovery-after-nodes"></span>
 
 Recovery after data nodes
 .........................
 
-The check for the :ref:`gateway.recover_after_data_nodes
-<gateway.recover_after_data_nodes>` verifies that the number of started nodes
-before the cluster starts must be greater than the half of the expected number
-of data nodes and equal to or less than number of data nodes in the cluster.
+This check looks at the 
+:ref:`gateway.recover_after_data_nodes <gateway.recover_after_data_nodes>` 
+setting and checks if its value is greater than half the configured expected 
+number, but not greater than the configured expected number.
 
 .. NOTE::
 
    For backward compatibility, setting the deprecated
    :ref:`gateway.recover_after_nodes <gateway.recover_after_nodes>` instead
-   is still supported. It must then be equal to less than the number data and
-   master nodes.
+   is still supported. 
 
 ::
 
   (E / 2) < R <= E
 
 Here, ``R`` is the number of :ref:`recovery <gloss-shard-recovery>` nodes and
-``E`` is the number of expected nodes.
+``E`` is the number of expected (data) nodes.
+
+If recovery is started when some nodes are down CrateDB will proceed on the
+basis the nodes that are down may not be coming back, and it will create new 
+replicas and rebalance shards as necessary, this is an expensive operation
```

Comment:
If nodes are just down for a short period of time, on could use the allocation settings?
Is this really such a problematic/common scenario, that we need to describe it?

This basically implies, that there is a very large time delay between starting the first nodes (after a down time) and some nodes joining. The creation of replica by default is quite throttled, both in terms of count and throughput.

Reply:
I made some changes to this paragraph, please take a look again to see if it looks better now. Thank you.